lib/tether/net_linux_test.go

```diff
@@ -69,7 +69,7 @@ func TestSetIpAddress(t *testing.T) {
 						Name: "bridge",
 					},
 					Default: true,
-					Gateway: gwIP,
+					Gateway: *gwIP,
```

Comment:
found this myself - no idea how this got committed as I always run the package tests directly as well as make test when making tether changes.